src/CONST.js

```diff
@@ -12,7 +12,7 @@ const CONST = {
     MODAL: {
         MODAL_TYPE: {
             CENTERED: 'centered',
-            BOTTOM_DOCKED: 'bottom_docked',
+            CREATE_MENU: 'create_menu',
```

Comment:
Not a problem!
I explained why I did this change in the PR description. My thought process was that maybe it would be nice to leave all the logic inside the `getModalStyle`, but I acknowledge it comes with the price of creating not really reusable styles.

I wanted to see what you think. I will push a new commit with the changes!

Reply:
Adding on my comment above, I also thought that the popover style is not really a reusable one, especially because of the absolute positioning values it has (provided by applying margin on `modalStyle`).

Reply:
Looking at the other way now, it really just ads one (or two) lines in the `CreateMenu` file. I changed my mind.

The initially proposed way makes more sense.